home/urls.py

```diff
@@ -10,5 +10,6 @@
     path('tags/', views.tags, name='tags'),
     path('explore/', QuestionsListView.as_view(), name='explore-page'),
     path('explore/new_question', views.new_question, name='new_question'),
-    path('login/', auth_views.LoginView.as_view(template_name='home/login.html'), name='login')
+    path('login/', auth_views.LoginView.as_view(template_name='home/login.html'), name='login'),
+    path('thumbs', views.thumbs, name='thumbs')
```

Comment:
Just to make sure I understand, you made a different path for thumbs so you could call the function in views for each answer, right?

Reply:
yes